python/cudf/cudf/io/parquet.py

```diff
@@ -712,9 +738,92 @@ def _get_partitioned(
     return full_paths, metadata_file_paths, grouped_df, part_offsets, filename
 
 
+@_cudf_nvtx_annotate
+def _get_partitioned_new(
```

Comment:
is this function supposed to supersede `_get_partitioned`? Can the other one be removed?

Reply:
Nope, it is just extracting the necessary logic that can be shared, renamed it now to `_get_groups_and_offsets`